pysat/_meta.py

```diff
@@ -138,7 +266,7 @@ def __repr__(self, recurse=True):
             for item_name in self.keys_nD():
                 output_str += '\n\n'
                 output_str += 'Metadata for '+item_name+'\n'
-                output_str += self._ho_data[item_name].__repr__(False)
+                output_str += self.ho_data[item_name].__repr__(False)
```

Comment:
What is ho_data?

Reply:
ho_data stands for higher order data. Clearly I should add a bit more to docstring.

Here is the issue. Consider how pysat stored superdarn data. For each slice of time there are a bunch of different variables stored as part of the drift vectors. All of this is stored under the single main header, 'vectors', or something like that. So, to get metadata on all those subvariables, use darn.meta['vectors'], which will return another Meta instance with all the subvariables. It is a recursive object. ho_data stores those additional Meta objects.

I had a commit which replaced that attribute ho_data with _ho_data . It broke the ICON IVM software and it was quicker to undo the commit in pysat for the moment. I think it may need to go back in.